src/components/redux/books/books.js

```diff
@@ -0,0 +1,28 @@
+const initialState = [];
+
```

Comment:
- [ ] Please define the `actionTypes` for adding and removing a book actions to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux).